src/Subscriptions/DataTransferObjects/SubscriptionQueryData.php

```diff
@@ -77,6 +77,11 @@ final class SubscriptionQueryData
      * @var int
      */
     public $donationFormId;
+    /**
+     * @unreleased
```

Comment:
Could we move this tag up to the class level?

Reply:
Resolved https://github.com/impress-org/givewp/pull/8126/commits/35f1c6e858ac768b1b35e75378e18843ebf1d2db